frontend/src/citizen-frontend/pedagogical-documents/PedagogicalDocuments.tsx

```diff
@@ -145,110 +147,151 @@ export default function PedagogicalDocuments() {
         )}
       </FixedSpaceRow>
     )
-  }
+  })
 
-  const PedagogicalDocumentsDisplay = ({
-    items
-  }: {
-    items: PedagogicalDocumentCitizen[]
-  }) => {
-    return (
-      <>
-        <Mobile>
-          <ContentArea opaque paddingVertical="L" paddingHorizontal="zero">
-            <PaddedDiv>
+  const PedagogicalDocumentsDisplay = React.memo(
+    function PedagogicalDocumentsDisplay({
+      items
+    }: {
+      items: PedagogicalDocumentCitizen[]
+    }) {
+      const moreThanOneChild =
+        items.reduce(
+          (childIds, doc) =>
+            childIds.includes(doc.childId)
+              ? childIds
+              : [...childIds, doc.childId],
+          [] as string[]
+        ).length > 1
+      return (
+        <>
+          <Mobile>
+            <ContentArea opaque paddingVertical="L" paddingHorizontal="zero">
+              <PaddedDiv>
+                <H1 noMargin>{t.pedagogicalDocuments.title}</H1>
+                <p>{t.pedagogicalDocuments.description}</p>
+              </PaddedDiv>
+              {items.length > 0 && (
+                <PedagogicalDocumentsList
+                  items={items}
+                  showChildrenNames={moreThanOneChild}
+                />
+              )}
+            </ContentArea>
+          </Mobile>
+          <Desktop>
+            <ContentArea opaque paddingVertical="L">
```

Comment:
Instead of adding (in this case useless) `useMemo` wrapper, these inner components should be moved to top level instead. See https://reactjs.org/docs/higher-order-components.html#dont-use-hocs-inside-the-render-method